src/components/global/Header/ConnectWalletButton.tsx

```diff
@@ -113,40 +138,136 @@ const WalletButton = () => {
+            <TokenModal.AddressName>
+              {addressHidden
+                ? obscureAddress(walletWeb3ReactContext.account)
+                : walletWeb3ReactContext.account}
+            </TokenModal.AddressName>
+            <TokenModal.ToggleAddress
+              onClick={() => setAddressHidden(!addressHidden)}
+              src={
+                addressHidden
+                  ? '/images/icon-eye-hide.png'
+                  : '/images/icon-eye-show.png'
+              }
+              alt={addressHidden ? 'Show Address' : 'Hide Address'}
+            />
+          </TokenModal.AddressRow>
+          <TokenModal.WalletActionsRow>
+            <TokenModal.WalletAction
+              onClick={() => {
+                navigator.clipboard.writeText(walletWeb3ReactContext.account)
+                setCopiedAddress(true)
+                setTimeout(() => setCopiedAddress(false), 2000)
+              }}
+            >
+              <img
+                src="/images/icon-copy.svg"
+                height="15px"
+                width="16px"
+                alt="Copy Address"
+              />
+              {copiedAddress ? `Copied!` : `Copy Address`}
+            </TokenModal.WalletAction>
+            <NextLink
+              href={`https://etherscan.io/address/${walletWeb3ReactContext.account}`}
+            >
+              <TokenModal.WalletAction>
+                <img
+                  src="/images/icon-export.svg"
+                  height="15px"
+                  width="15px"
+                  alt="Etherscan"
+                />
+                Etherscan
+              </TokenModal.WalletAction>
+            </NextLink>
+            <TokenModal.WalletAction
+              onClick={() => walletWeb3ReactContext.deactivate()}
+            >
+              <img
+                src="/images/icon-wallet.svg"
+                height="15px"
+                width="15px"
+                alt="Change Wallet"
+              />
+              Change Wallet
+            </TokenModal.WalletAction>
+          </TokenModal.WalletActionsRow>
+          <TokenModal.BigRow>
+            <div>Bankless DAO / $BANK</div>
+            <TokenModal.BigBank>
+              <strong>{whaleBalance}</strong>
+              <img
+                onMouseEnter={() => {
+                  setTokenEffect(true)
+                }}
+                onMouseLeave={() => {
+                  setTokenEffect(false)
+                }}
+                src={
+                  tokenEffect
+                    ? '/images/token-3d-animated.gif'
+                    : '/images/token-3d.png'
+                }
+                height="85px"
+                width="85px"
+                alt="3d token icon"
+              />
+            </TokenModal.BigBank>
+            <TokenModal.BankBalance>
+              {commaBalance} <strong>BANK</strong>
+            </TokenModal.BankBalance>
+            {rawClaim ? (
+              <TokenModal.UnclaimedNotice
```

Comment:
Local testing note: You can change this `rawClaimed` to test the "Unclaimed Tokens" notice